src/components/assistant-code-block.tsx

```diff
@@ -70,6 +89,27 @@ const LanguageBlock = ( props: ContextProps & CodeBlockProps ) => {
 						} );
 					} }
 				></CopyTextButton>
+				{ isBlueprint && (
+					<Button
+						icon={ external }
```

Comment:
What if we used `create` (`+`) icon for the button - since clicking it will lead to new site creation?